test/cql-pytest/test_tombstone_limit.py

```diff
@@ -346,3 +346,23 @@ def test_empty_table(cql, test_keyspace, lowered_tombstone_limit, driver_bug_1):
         assert list(cql.execute(f"SELECT * FROM {table}")) == []
         assert list(cql.execute(f"SELECT * FROM {table} WHERE pk = 0")) == []
         assert list(cql.execute(f"SELECT * FROM {table} WHERE v = 0 ALLOW FILTERING")) == []
+
+
+# Unpaged query should be not affected
+def test_unpaged_query(cql, table, lowered_tombstone_limit, driver_bug_1):
```

Comment:
Wanted to say the config item description needs to be adjusted, but it already anticipated this fix:

```
db/config.cc-   , query_tombstone_page_limit(this, "query_tombstone_page_limit", liveness::LiveUpdate, value_status::Used, 10000,
db/config.cc-        "The number of tombstones after which a query cuts a page, even if not full or even empty."
```

Reply:
Ugh the name even says "page limit".